src/main/resources/templates/db_scripts/schema.json

```diff
@@ -499,6 +503,10 @@
           "fieldName": "locationId",
           "caseSensitive": false
         },
+        {
+          "fieldName": "holdingId",
+          "caseSensitive": false
+        },
```

Comment:
Now used when doing unopen order and update piece that has flag deleteHolding

Before change:
`15:55:10 [269569/orders;749385/orders-storage] [diku] [7d5a86d0-f6d7-5826-9b6b-074d323d9437] [mod_orders_storage] WARN  CQL2PgJSON           Doing LIKE search without index for pieces.jsonb->>'holdingId', CQL >>> SQL: holdingId == f50acfe3-cb60-4fdb-b711-428c60527191 >>> lower(f_unaccent(pieces.jsonb->>'holdingId')) LIKE lower(f_unaccent('f50acfe3-cb60-4fdb-b711-428c60527191'))
`